src/main/java/net/sf/jabref/gui/dbproperties/SaveActionsPanel.java

```diff
@@ -138,19 +134,22 @@ public void storeSettings(MetaData metaData) {
     }
 
     public boolean hasChanged() {
-        List<FieldFormatterCleanup> newActions = ((SaveActionsListModel) actionsList.getModel()).getAllActions();
+        List<FieldFormatterCleanup> newActions = ((SaveActionsListModel<FieldFormatterCleanup>) actionsList.getModel())
```

Comment:
Do you need to specify `FieldFormatterCleanup` on the right hand side or is it enough to write `<>`? I have no idea...